src/js/background/identityState.js

```diff
@@ -54,9 +106,57 @@ const identityState = {
     return this.storageArea.set(cookieStoreId, containerState);
   },
 
+  async updateUUID(cookieStoreId, uuid) {
+    if (cookieStoreId && uuid) {
+      const containerState = await this.storageArea.get(cookieStoreId);
+      containerState.macAddonUUID = uuid;
+      await this.storageArea.set(cookieStoreId, containerState);
+      return uuid;
+    } 
+    throw new Error ("cookieStoreId or uuid missing");
```

Comment:
I personally would make this the other way around:
```
if (!cookieStoreId || !uuid) {
  throw new Error ("cookieStoreId or uuid missing");
}
```
It has the benefit of reducing nesting in functions returning earlier like that.